### DISCUSSION TITLE
How can open-source version of 4playerchess.com be designed with Stockfish?

### DISCUSSION THREAD

**Discussion by PrasadNR** [official-stockfish/Stockfish#3843] [Ideas]
I am not sure if this has to be started from "scratch" with reinforcement deep-learning or min-max alpha-beta search-tree pruning or other algorithms.

I am also not sure if Stockfish can be modified slightly (in order to accommodate for additional 4 x 8 x 3 cells given that Stockfish works for 8 x 8 cells from what I know).

**Comment by vondele** [comment]
unlikely to be easy to do using stockfish, which is specialized for standard chess. There is a fork of SF dealing with variants https://github.com/ianfab/Fairy-Stockfish but I don't know what can be easily supported in that case.
